pkg/server/server.go

```diff
@@ -151,7 +151,14 @@ func (s *Server) handle(apiOp *types.APIRequest, parser parse.Parser) {
 	} else if code > http.StatusOK {
 		apiOp.Response.WriteHeader(code)
 	}
-	metrics.RecordResponseTime(apiOp.Schema.ID, apiOp.Method, strconv.Itoa(code), float64(time.Since(requestStart).Milliseconds()))
+
+	var schemaID string
+	if apiOp.Schema != nil {
+		schemaID = apiOp.Schema.ID
+	} else {
+		schemaID = "unknown"
+	}
+	metrics.RecordResponseTime(schemaID, apiOp.Method, strconv.Itoa(code), float64(time.Since(requestStart).Milliseconds()))
```

Comment:
I wonder if it's worth recording the response time at all for an invalid URL path, @rmweir what do you think?